SB2018/2018-01-12.md

```diff
@@ -0,0 +1,34 @@
+---
+to: All teams
+subject: SourceBots Mentoring and Tech Days
+---
+
+Hi all,
+
+We hope you're progressing well on your robot!
+Some important things to announce so keep reading:
```

Comment:
This sentence is not only missing a comma, but i'm not sure it's necessary, or should at least be worded differently and put on the previous lines